### ISSUE TITLE
Dockerfile for ease of installation?

### ISSUE THREAD

**Issue by vielmetti** [CulturePlex/Sylva#35]
Is there a SylvaDB Dockerfile that handles all of the building of all of the dependencies and setting up of the environment for this system?

**Comment by versae** [comment]
There is a partial one with basic functionality: no analytics or reports. If you are interested I can clean it up a bit and send it to you.

**Comment by vielmetti** [comment]
Yes, this would be interesting - I'm mostly looking to do something that gives me a quick start to the system to assess it (and thus a few missing features are no big deal)
